nipoppy/workflow/catalog.py

```diff
@@ -11,13 +11,14 @@
     load_status,
 )
 
-def read_and_process_status(status_csv, session_id, logger):
+def read_and_process_status(status_csv, session_id=None, logger=None):
```

Comment:
Why would `session_id` ever be `None`?

Reply:
The bug that this PR is supposed to fix is due to `run_bids_conv.py` loading the doughnut for only a single session, then updating it and saving it (which makes it lose all the rows with other sessions). But I just realized that I should have used `nipoppy.workflow.utils.load_status()` instead 😅